api/swagger.yaml

```diff
@@ -506,6 +506,9 @@ definitions:
       Retries:
         description: "The number of consecutive failures needed to consider a container as unhealthy. 0 means inherit."
         type: "integer"
+      StartPeriod:
+        description: "Start period for the container to initialize before starting health-retries countdown in nanoseconds. It should be 0 or not less than 1000000000(1s). 0 means inherit."
```

Comment:
> It should be 0 or not less than 1000000000(1s).

Any specific reason for this constraint? I think `StartPeriod` can be any non-negative value.